roles/hosted_engine_setup/README.md

```diff
@@ -356,6 +362,10 @@ In order to proceed with the deployment, before deleting the lock-file, make sur
 
 Both of the lock-file path and the engine's URL will be presented during the role execution.
 
+**On Failure**
+
+If "Add Host" failed and left the host in status "non_operational", by default the deployment will be paused, similarly to "Manual" above, so that the user can try to fix the host to get it to "up" state, before removing the lock file and continuing. If you want the process to fail instead of pausing, set he_pause_after_failed_add_host to false.
```

Comment:
Missig backticks for `he_pause_after_failed_add_host`

Reply:
Done